### PR TITLE
button will now add new task to list

### LINKED ISSUE #3: Add a static list of todos using ul / li

### PR DESCRIPTION
**Pull request by harshilvithlani**
fixes #3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,20 +2,55 @@
...
-    <h1>My Todo app</h1>
-    <form>
-    	<input type="Text" name="EnterTask" value="EnterTask">
-    	
-    	<input type="button" name="Submit" id="Submit" value="submit">
+  <div id="myDIV" class="header">
+  <h2>My To Do List</h2>
+  <input type="text" id="myInput" placeholder="Title..."> <br>
+
+  <input type="button" name="Add New Task" value="Add New Task" span onclick="newElement()" class="addBtn">
```
Comment (lakhansamani) [inline index.html]: this should be self closing tag

Comment (lakhansamani) [reply index.html]: no need to use `span`

Comment (lakhansamani) [reply index.html]: we should use`type="submit"` so that on press of enter also a new task can be added.
You can use `event.preventDefault()` to stop refresh on submit

File: index.html [hunk 2]
```diff
@@ -2,20 +2,55 @@
...
   </head>
   <body>
-    <h1>My Todo app</h1>
-    <form>
-    	<input type="Text" name="EnterTask" value="EnterTask">
-    	
-    	<input type="button" name="Submit" id="Submit" value="submit">
+  <div id="myDIV" class="header">
+  <h2>My To Do List</h2>
+  <input type="text" id="myInput" placeholder="Title..."> <br>
```
Comment (lakhansamani) [inline index.html]: `input` and `br` are self closing tag

File: index.html [hunk 3]
```diff
@@ -2,20 +2,55 @@
...
+  		var span = document.createElement("SPAN");
+  		var txt = document.createTextNode("\u00D7");
+  		span.className = "close";
+  		span.appendChild(txt);
+  		li.appendChild(span);
+
+  		for (i = 0; i < close.length; i++)
+  		{
+    		close[i].onclick = function() 
+    	{
```
Comment (lakhansamani) [inline index.html]: indentation is off here.

**Review by lakhansamani — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (harshilvithlani) [reply index.html]: > this should be self closing tag

which tag?

File: index.html [hunk 4]
```diff
@@ -2,20 +2,58 @@
...
   </head>
   <body>
-    <h1>My Todo app</h1>
-    <form>
-    	<input type="Text" name="EnterTask" value="EnterTask">
-    	
-    	<input type="button" name="Submit" id="Submit" value="submit">
+  <div id="myDIV" class="header">
+  <h2>My To Do List</h2>
+  <input type="text" id="myInput" placeholder="Title..."/> <br/>
```
Comment (lakhansamani) [inline index.html]: Why did you remove `form` tag?

File: index.html [hunk 5]
```diff
@@ -2,20 +2,58 @@
...
+
+  <input type="submit" name="Add New Task" value="Add New Task" onclick="newElement()" class="addBtn"/>
```
Comment (lakhansamani) [inline index.html]: this event should be part of `<form onsubmit="newElement()">`

**Review by lakhansamani — CHANGES_REQUESTED** [review]

**Review by lakhansamani — APPROVED** [review]
